lib/marklogic.js

```diff
@@ -369,6 +371,42 @@ function MarkLogicClient(connectionParams) {
 
   this.logger = null;
 }
+
+/**
+ * Tests if a connection is successful.
+ * @method DatabaseClient#testConnection
+ * @since 2.1
+ * @returns true if the connection is successful and false otherwise with an error message
```

Comment:
Would it be more accurate to say that it returns an object with a connected property of true or false and, in the false case, httpStatusCode and httpStatusMessage properties identifying the failure?